gen_docs.sh

```diff
@@ -1,5 +1,6 @@
-#! /bin/bash
+#!/bin/bash
 
+scripts/extract_bp_help
```

Comment:
You should be putting this generated doc directly into ./doc/ then it can be part of the main doc rendering. Don't put it in the repo root.